ls1intum/Artemis
# `Programming exercises`: Fix duplicate build job trigger for local continuous integration

### Checklist
#### General


#### Server

### Motivation and Context


This PR fixes the bug described  #7517. 
### Description
Removed the line where the build gets triggered the second time. Adapted some tests.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student

1. As a student, submit to a new programming exercise and wait for result
2. As instructor, check the submissions for the exercise: Submission for the student should only have one result

### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Review Progress






#### Code Review
#### Manual Tests
